src/applications/financial-status-report/utils/helpers.js

```diff
@@ -12,6 +12,10 @@ export const fsrFeatureToggle = state => {
   return toggleValues(state)[FEATURE_FLAG_NAMES.showFinancialStatusReport];
 };
 
+export const medicalCopaysFeatureToggle = state => {
```

Comment:
@craigwheeler I can leave this in here so you can copy it over to the medical copays app if you want otherwise I can just delete it.